tree/tree/test/BulkApiVarLength.cxx

```diff
@@ -96,14 +96,14 @@ TEST_F(BulkApiVariableTest, stdRead)
          ASSERT_TRUE(false);
       }         
       if (R__unlikely(myF.GetSize() != static_cast<size_t>(ev % 10))) {
-         printf("Incorrect number of entries on float branch: %lu, expected %lld (event %lld)\n",
+         printf("Incorrect number of entries on float branch: %zu, expected %lld (event %lld)\n",
                 myF.GetSize(),
                 ev % 10,
                 ev);
          ASSERT_TRUE(false);
       }
       if (R__unlikely(myD.GetSize() != static_cast<size_t>(ev % 10))) {
-         printf("Incorrect number of entries on double branch: %lu, expected %lld (event %lld)\n",
+         printf("Incorrect number of entries on double branch: %zu, expected %lld (event %lld)\n",
```

Comment:
I am 'not' sure whether %zu is available on Windows.  (@bellenot ?)

Reply:
I found next: MS Visual Studio didn't support %zu printf specifier before VS2013. Starting from VS2013 (e.g. _MSC_VER >= 1800) %zu is available.

Reply:
Can you just use %lu and a cast?

Reply:
Done!